restart/EXTNET-GCONINJE-VREPW.DATA

```diff
@@ -0,0 +1,344 @@
+DISGAS
+VAPOIL
+
+METRIC
+
+START
+ 30  'NOV' 2018 /
+
+NSTACK
+  50 /
+
+--
+GRIDOPTS
+ 'YES'	      0 /
+
+EQLDIMS
+ 2 1*  25 /
+
+
+EQLOPTS
+ THPRES  /
+
+
+REGDIMS
+-- max. ntfip  nmfipr  max. nrfreg   max. ntfreg
+   2          1       1*            2    /
+
+--
+WELLDIMS
+--max.well  max.con/well  max.grup  max.w/grup
+ 10	    15 	          3	     10   /
+
+--
+TABDIMS
+--ntsfun     ntpvt  max.nssfun  max.nppvt  max.ntfip  max.nrpvt
+  10          1      50          60         2         60 /
+
+
+VFPPDIMS
+--max.rate  max.THP   max.fw   max.fg	max.ALQ    max.tabs
+  40	    20        20       20	 13	    60	    /
+
+--
+FAULTDIM
+120 / max. number os fault segments
+
+NETWORK
+  5  5  /
+
+UNIFIN
+UNIFOUT
+
+------------------------------------------------------------------------------------------------
+GRID
+------------------------------------------------------------------------------------------------
+
+--
+NEWTRAN
+
+--
+GRIDFILE
+0  1/
+
+--
+GRIDUNIT
+METRES  /
+
+--
+INIT
+
+
+--
+
+
+
+INCLUDE
+ '../network/include/MODEL02-GRID.inc' /
+
+INCLUDE
+ '../network/include/MODEL02-FLUXNUM.inc' /
+
+
+INCLUDE
+ '../network/include/MODEL02-PORO.inc' /
+
+INCLUDE
+ '../network/include/MODEL02-PERMX.inc' /
+
+INCLUDE
+ '../network/include/MODEL02-NTG.inc' /
+
+
+COPY
+ PERMX PERMY /
+/
+
+
+INCLUDE
+ '../network/include/MODEL02-PERMZ.inc'/
+
+
+RPTGRID
+ 'ALLNNC' /
+
+------------------------------------------------------------------------------------------------
+EDIT
+------------------------------------------------------------------------------------------------
+
+
+------------------------------------------------------------------------------------------------
+PROPS
+------------------------------------------------------------------------------------------------
+
+NOECHO
+
+INCLUDE
+ '../network/include/MODEL02-PVT.inc' /
+
+INCLUDE
+ '../network/include/MODEL02-ROCK.inc' /
+
+INCLUDE
+ '../network/include/MODEL02-SWOF.inc' /
+
+INCLUDE
+ '../network/include/MODEL02-SGOF.inc' /
+
+
+------------------------------------------------------------------------------------------------
+REGIONS
+------------------------------------------------------------------------------------------------
+
+INCLUDE
+ '../network/include/MODEL02-EQLNUM.inc' /
+
+INCLUDE
+ '../network/include/MODEL02-FIPNUM.inc' /
+
+INCLUDE
+ '../network/include/MODEL02-SATNUM.inc' /
+
+------------------------------------------------------------------------------------------------
+SOLUTION
+------------------------------------------------------------------------------------------------
+
+
+RPTRST
+  'BASIC = 2' 'PBPD' /
+
+EQUIL
+-- Datum    P     woc     Pc   goc    Pc  Rsvd  Rvvd
+ 2561.59  268.55  2645.21   0.0 2561.59  0.0   1   0   0 /
+ 2584.20  268.71 2685.21   0.0 2584.20  0.0   5   0   0 /
+
+RSVD
+ 2561.59  122.30
+ 2597.0  110.00
+ 2660.7  106.77
+ 2697.0  106.77 /
+
+ 2584.20  122.41
+ 2599.9  110.00
+ 2663.6  106.77
+ 2699.9  106.77 /
+
+
+THPRES
+ 1  2 /
+/
```

Comment:
If this is supposed to be a restart case, then I would expect to find the keyword `RESTART` here in the `SOLUTION` section instead of an `EQUIL`-based initialisation.  If, on the other had, it is **not** intended to be a restart case, then the `SKIPREST` keyword should typically not be in the `SCHEDULE` section.

Reply:
I can see that being confusing, this is intended as the base run - it is just my personal preference to always include SKIPREST since it has no effect in the base run and I tend to forget it first time around if setting up the restart manually.. :)   Can certainly remove it if you prefer?

Reply:
> it is just my personal preference to always include `SKIPREST` since it has no effect in the base run and I tend to forget it first time around if setting up the restart manually

Ah, okay&ndash;understood.

> Can certainly remove [`SKIPREST`] if you prefer?

Nah, it's fine, but maybe just add a comment before `SKIPREST` in the `.DATA` file that the keyword is intended, but harmless, in a base run?